dashboard/test/models/census/census_submission_form_map_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class Census::CensusSubmissionFormMapTest < ActiveSupport::TestCase
+  test "empty census submission form map" do
```

Comment:
nit: add expected result to the test case name, e.g. `empty census submission form map is invalid`

Reply:
Updated